src/config/key.rs

```diff
@@ -119,6 +119,7 @@ pub fn parse_key(input: &str) -> Result<Key, Box<dyn Error>> {
             REL_WHEEL_HI_RES = 0x0b,
             REL_HWHEEL_HI_RES = 0x0c,
         */
+        "ANY" => Key(DISGUISED_EVENT_OFFSETTER + 26),
```

Comment:
26 feels like a magic number at a glance. Could you make a constant that hides this implementation detail, add a clarifying comment above the constant, and use the constant here?

Reply:
I've updated the PR to use a constant. I hope I understood correctly what you meant.